pydicom/dataset.py

```diff
@@ -330,6 +342,16 @@ class Dataset(dict):
     """
     indent_chars = "   "
 
+    # TODO: this the correct list?
+    _BINARY_VR_VALUES = ['OW', 'OB', 'OD', 'OF', 'OL', 'UN',
+                         'OW/OB', 'OW or OB', 'OB or OW', 'US or SS']
+
```

Comment:
I think 'OW/OB' and 'OW or OB' are obsolete (have been used in some older version of the DICOM dictionary), and 'US or SS' probably belongs to the int values.